packages/api/__tests__/specs/comment.ts

```diff
@@ -0,0 +1,55 @@
+import {MongoDBAdapter} from '@wepublish/api-db-mongodb'
```

Comment:
Is there a PR that will introduce some tests? Otherwise this file doesn't make any sense.

Reply:
Yes I will have the UT in another PR tonight

Reply:
@roosnic1 Yes I will have the UT in another PR tonight